installer/src/config/resources/unix_shortcut_specification.xml

```diff
@@ -2,19 +2,19 @@
 	<defaultCurrentUser/>
 	<programGroup defaultName=""
 		location="applications"
-		iconFile="$INSTALL_PATH/studio/48-jbds_icon.png"
+		iconFile="$INSTALL_PATH/studio/48-devstudio.icon.png"

```

Comment:
I renamed everything to ##-devstudio.icon.png for consistency. I left the underscore for the uninstaller though. Could change that if you want so uninstaller and other icons all use "." instead of "_".